src/renderer/views/RegisterView/subviews/ActivationKeySubview.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import ActivationKeyForm, {
+  FormData,
+} from "src/renderer/components/ActivationKeyForm";
+import H1 from "src/renderer/components/ui/H1";
+import { ExtLink } from "src/renderer/components/ui/Link";
+import { trackEvent } from "src/utils/mixpanel";
+import { useStore } from "src/utils/useStore";
+import { RegisterState } from "..";
+
+interface Props {
+  setState: (state: RegisterState) => void;
+}
+
+function ActivationKeySubview({ setState }: Props) {
+  const accountStore = useStore("account");
+
+  const onSubmit = async ({ activationKey }: FormData) => {
+    trackEvent("Launcher/EnterActivationCode");
+
+    setState("waitActivation");
+
+    accountStore.setActivationKey(activationKey);
+  };
+
+  return (
+    <>
+      <H1>Activate your address</H1>
+      <p style={{ marginBlockEnd: 54 }}>
+        You need an activation code to activate your Nine Chronicles address. If
+        you already have one, you can paste it below and activate it now.
+      </p>
+      <ExtLink target="_blank" href="">
+        Get the code
+      </ExtLink>
```

Comment:
Link for `Get the code` is missing

Reply:
also, It may be useful if the address is configurable.

Reply:
Previously I made ActivationCodeLink entry in config.json to get this link as I remember, I guess we can use same method as before.

Reply:
Made the link to read from the config file.